media/css/mozorg/webvision.scss

```
@@ -0,0 +1,122 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+$font-path: '/media/fonts';
+$image-path: '/media/protocol/img';
+
+@import '~@mozilla-protocol/core/protocol/css/includes/lib';
+@import '~@mozilla-protocol/core/protocol/css/components/sidebar-menu';
+@import '~@mozilla-protocol/core/protocol/css/templates/main-with-sidebar';
+
+.c-webvision-article {
+    h2 {
+        @include text-title-sm;
+    }
+
+    h3 {
+        @include text-title-xs;
+    }
+
+    h4 {
+        @include text-title-2xs;
+    }
+
+    h1,
+    h2,
+    h3,
+    h4 {
+        a {
+            position: relative;
+            transition: 2s;
+            visibility: hidden;
+
+            &::after {
+                content: '#';
+                color: $color-marketing-gray-30;
+                position: absolute;
+                left: 5px;
+                transition: all 0.2s;
```

Nit: Since `color` is the only property transitioning you can declare it explicitly. And this is just a personal style thing, but I like milliseconds for timing, though decimals are just as valid.
```suggestion
                transition: color 200ms;
```